contracts/AssetPool.sol

```diff
@@ -72,13 +77,37 @@ contract AssetPool is Ownable, IAssetPool {
         uint256 timestamp
     );
 
+    event ApprovedAssetPoolUpgrade(address newAssetPool);
     event AssetPoolUpgraded(
         address indexed underwriter,
         uint256 collateralAmount,
         uint256 covAmount,
         uint256 timestamp
     );
-    event ApprovedAssetPoolUpgrade(address newAssetPool);
+
+    event WithdrawalDelayUpdateStarted(
+        uint256 withdrawalDelay,
+        uint256 timestamp
+    );
+    event WithdrawalDelayUpdated(uint256 withdrawalDelay);
+    event WithdrawalTimeoutUpdateStarted(
+        uint256 withdrawalTimeout,
+        uint256 timestamp
+    );
+    event WithdrawalTimeoutUpdated(uint256 withdrawalTimeout);
+
+    modifier onlyAfterWithdrawalGovernanceDelay(
```

Comment:
Non-blocking (#https://github.com/keep-network/coverage-pools/issues/82): We have all other modifiers documented. Can we document this one as well?

Reply:
https://github.com/keep-network/coverage-pools/pull/105/commits/c0764c44ac0bc0884c60130ce06bce5fefa46cb8